pkg/ngrok.go

```diff
@@ -41,6 +53,33 @@ func RunNgrok(ctx context.Context) {
 	}
 }
 
+func init() {
```

Comment:
```suggestion
func init() {
```
is this function being used? i didnt see a reference in your code?

Reply:
how often is this failing? with all the hard work of cleaning up logs and updating the logger to produce better output is this going to produce 10 times of unnecessary noise?

Reply:
@jarededwards you are right, forgot to remove, done! thanks!